src/test/java/com/purbon/kafka/topology/integration/TopicManagerIT.java

```diff
@@ -57,9 +53,7 @@
 
   @BeforeClass
   public static void setup() {
-    container =
-        new KafkaContainer(
-            TestcontainersConfiguration.getInstance().getKafkaDockerImageName().withTag("5.5.0"));
+    container = new SaslPlaintextKafkaContainer();
```

Comment:
This was using earlier 5.5.0, while I love to change all to use 6.0 I would love to have it uniform. Can you make sure this is kept consistent across the PR?

Reply:
I'm not sure I understand... do you want all Kafka (test)containers to use 6.0.0, or do you want me to rewrite the old docker-compose scripts too?

Reply:
lets leave the docker-compose alone. But in your addition, can we use version 5.5.0 ? later I would like to add a parameter in maven where people can pass the version we aim to run the test with.

Make sense?

Reply:
Yes, makes sense. But it will have to be another day, because I'm long overdue preparing Friday night dinner for my wife, and just got lost in a rebasing nightmare. Sorry.

Reply:
man, no pressure at all! a happy wife is the most important thing in life;-P